wcc-contentful/lib/wcc/contentful/middleware/store.rb

```diff
@@ -88,4 +84,28 @@ def select?(_entry)
   def transform(entry)
     entry
   end
+
+  class Query < WCC::Contentful::Store::Base::Query
```

Comment:
This finally, actually, fixes the error we were having another time due to the middleware and `find_all`.